packages/contracts/src/dollar/mocks/MockUbiquistick.sol

```diff
@@ -0,0 +1,15 @@
+// contracts/GLDToken.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+
+contract MockUbiquistick is ERC1155 {
+    // cspell: disable-next-line
+    constructor() ERC1155("ubiquiStick") {
```

Comment:
```suggestion
    constructor() ERC1155("UbiquiStick") {
```

Reply:
Will make a commit for this. I don't want conventional commit to fail.

Reply:
I'm pretty sure that @Steveantor updated our conventional commits to support suggestions from the GitHub reviews process!

Reply:
> I'm pretty sure that @Steveantor updated our conventional commits to support suggestions from the GitHub reviews process! 

Just accepted the commit.